policy/modules/services/spamassassin.te

```diff
@@ -203,13 +213,17 @@ stream_connect_pattern(spamc_t, { spamd_runtime_t spamd_tmp_t }, { spamd_runtime
 kernel_read_kernel_sysctls(spamc_t)
 kernel_read_system_state(spamc_t)
 
-corenet_all_recvfrom_netlabel(spamc_t)
-corenet_tcp_sendrecv_generic_if(spamc_t)
-corenet_tcp_sendrecv_generic_node(spamc_t)
-corenet_udp_bind_generic_node(spamc_t)
+tunable_policy(`spamassassin_can_network',`
```

Comment:
This block needs to move down after the `use_samba_home_dirs` block.

Reply:
Yes, I'll move the block.

Reply:
There seems to be two copies of this block.

Reply:
I didn't remove it by mistake, thanks for telling me. It has been fixed now !